spec/post_spec.rb

```diff
@@ -0,0 +1,95 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  before :each do
+    @user = User.new(name: 'Henok', posts_counter: 0)
+  end
+
+  # Unit tests for title
+  it '.valid should be true when title is present' do
+    post = Post.new(author: @user, title: 'Title')
+    expect(post).to be_valid
+  end
+
+  it '.valid should be false when title is not present' do
+    post = Post.new(author: @user)
+    expect(post).to_not be_valid
+  end
+
```

Comment:
- [ ] Great job for adding the test for User and post models but I think you forgot to test the comment and like model since it is mentioned in the requirements to test all the methods in the models so you need to test update_comments_counter and update_likes_counter in comment and like models so kindly add tests for them ⬇💪🏻🔥✅

![image](https://github.com/henokkhm/microverse-rails-blog/assets/58553711/de49979b-2dfe-4280-b97f-2a4eea8a24d8)